appserver/ejb/ejb-http-remoting/endpoint/src/main/java/fish/payara/ejb/http/protocol/LookupResponse.java

```diff
@@ -0,0 +1,22 @@
+package fish.payara.ejb.http.protocol;
+
+import java.io.Serializable;
+
+import javax.ejb.Stateful;
+import javax.ejb.Stateless;
+
+public class LookupResponse implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    public final String typeName;
+    public final String kind;
+
+    public LookupResponse(Class<?> ejbInterface) {
+        this.typeName = ejbInterface.getName();
+        this.kind = ejbInterface.isAnnotationPresent(Stateful.class) 
```

Comment:
This is not that easy, as currently the annotation can also be present on the class itself.

It will be up to the application code to decide the kind, for now, just always return "Stateless"